### PR TITLE
Set up postgres 

### PR DESCRIPTION
**Pull request by ebayasgalan**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -56,16 +56,19 @@ class App extends React.Component {
...
-      .then((res) => {
-        products = res.data;
-      })
-      .then(() => axios.get(`/api/wishlist/${productId}`))
       .then((res) => {
         this.setState({
-          products,
-          wishList: res.data.products,
-        });
+          products: res.data.rows
```
Comment (tdomingo) [inline client/src/components/App.jsx:61]: I think best practice would be sending the .rows data in your response, vs. filtering it out in the client, especially if your client can't use/doesn't need to see the other data.

File: schema.sql [hunk 2]
```diff
@@ -1,50 +1,48 @@
...
+    REFERENCES products (product_id)
+);
+
+CREATE INDEX ON tags (product_id);
 
 COPY tags (tag, product_id)
-FROM '/Users/Taivnaa/Desktop/hack-reactor/SDC/similar-products/tags.csv'
+FROM '/Users/Taivnaa/Desktop/hack-reactor/SDC/similar-products/database/tags.csv'
 DELIMITER ','
 CSV HEADER;
```
Comment (tdomingo) [inline schema.sql:48]: newline :)

File: client/src/components/product-components/Product.jsx [hunk 3]
```diff
@@ -113,41 +113,41 @@ class Product extends React.Component {
 
   render() {
     const { currentVariation, mouseOn } = this.state;
```
Comment (tdomingo) [inline client/src/components/product-components/Product.jsx:115]: it looks like currentVariation has been commented out of state - is this causing breakage in this component as a result?

**Review by tdomingo — APPROVED** [review]
Looks good overall! I would consider relocating some of the data re-shaping to the server vs. some of the client-side alterations that have been made, but I think the end result is good!

File: client/src/components/product-components/Product.jsx (hunk 3 above)
Comment (ebayasgalan) [reply client/src/components/product-components/Product.jsx:115]: No, it's not causing any breakage. Modified to have the backend expect same shape of response data

**Pull request merged**